mobile/lib/modules/activities/widgets/activity_tile.dart

```diff
@@ -106,7 +106,7 @@ class _ActivityAssetThumbnail extends StatelessWidget {
       decoration: BoxDecoration(
         borderRadius: const BorderRadius.all(Radius.circular(4)),
         image: DecorationImage(
-          image: ImmichImage.remoteThumbnailProviderForId(assetId),
+          image: ImmichRemoteImageProvider(assetId: assetId),
```

Comment:
`isThumbail: true` is needed here since the previous implementation was only requesting a thumbnail

Reply:
Nice catch! Got it.